README.md

```diff
@@ -66,6 +66,12 @@ Initial TS Compilation For Monorepo Packages
 yarn typecheck
 ```
 
+Adding a package to the codebase can be done with Lerna
```

Comment:
That new link should probably go here instead

Reply:
Thanks for the feedback!